hazelcast/test/src/HazelcastTests8.cpp

```diff
@@ -2316,6 +2317,21 @@ TEST_F(connection_manager_translate, default_config_uses_private_addresses)
 {
     ASSERT_FALSE(client_config().get_network_config().use_public_address());
 }
+
+TEST_F(connection_manager_translate,
+       if_remote_adress_provider_cannot_translate_adress_translate_should_throw_an_exception)
+{
+    auto client = new_client().get();
+    spi::ClientContext ctx(client);
+    connection::ClientConnectionManagerImpl connection_manager(
+      ctx,
+      std::unique_ptr<connection::AddressProvider>(
+        new spi::impl::discovery::remote_address_provider{
+          []() { return std::unordered_map<address, address>{}; }, true }));
+
+    EXPECT_THROW(connection_manager.get_or_connect(hazelcast::client::member{}),
```

Comment:
can you put a dummy address instead of empty constructor to be more realistic?

Reply:
Fixed.